BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -36,6 +36,13 @@ public final class BlueprintView: UIView {
 
     private let rootController: NativeViewController
 
+    public var environment: Environment {
```

Comment:
Ah ha, yay! I'd been wanting this for a while.

Reply:
Yeah me too - added a test as well